src/pages/Selected/components/UserInfo.jsx

```diff
@@ -1,43 +1,231 @@
+    )
+      .then(res => res.json())
+      .then(data => userInfo.setValues({ ...userInfo.info, ...data[0] }));
+  }, []);
+
+  const handlePhoneCheckBtn = e => {
+    e.preventDefault();
+    let s = parseInt(seconds);
+    let m = parseInt(minutes);
+    timer.current = setInterval(() => {
+      if (s > 0) {
+        setSeconds(--s);
+      }
+      if (s === 0) {
+        if (m === 0) {
+          clearInterval(timer.current);
+        } else {
+          setMinutes(--m);
+          setSeconds(59);
+          s = 59;
+        }
+      }
+    }, 1000);
+
+    fetch(`${API.verifi}`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization: localStorage.getItem('token'),
+      },
+      body: JSON.stringify({
+        mobile: phone,
+      }),
+    });
+  };
+
+  const handleSubmitCode = e => {
+    e.preventDefault();
+    clearInterval(timer.current);
+    setMinutes(5);
+    setSeconds(0);
+
+    fetch(`${API.verifi}/code`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization: localStorage.getItem('token'),
+      },
+      body: JSON.stringify({
+        mobile: phone,
+        code: phoneCheck,
+        sentDate: dateFormat(date),
+      }),
+    })
+      .then(res => res.json())
+      .then(data => alert(JSON.stringify(data.message)));
+  };
+
+  const handleSubmitInfo = e => {
+    e.preventDefault();
+    fetch(`${API.order}`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization: localStorage.getItem('token'),
+      },
+      body: JSON.stringify({
+        orderName: userInfo.info.testName,
+        totalAmount: 3000,
+        testId: seatId,
+        seatInfo: testSeats,
+      }),
+    })
+      .then(res => res.json())
+      .then(data => {
+        loadTossPayments(clientKey).then(tossPayments => {
+          tossPayments.requestPayment('카드', data).catch(error => {
+            if (error.code === 'USER_CANCEL') {
+              alert('결제창이 닫혔습니다.');
+            } else if (error.code === 'INVAILD_CARD_COMPANY') {
+              alert('유효하지않은 카드입니다');
+            }
+          });
+        });
+      });
+  };
+
+  const isPhoneVaild = phone.length >= 9;
+  const codeUnderSix = phoneCheck.length < 6;
+  const codeOverTen = phoneCheck.length > 10;
+  const isEmailVaild = email.includes('@');
+  const isAllChecked =
+    testSeats !== '' &&
+    name !== '' &&
+    isPhoneVaild &&
+    phoneCheck !== '' &&
+    isEmailVaild;
+
+  const makeWarning = id => {
+    if (id === 'phone') {
+      return !isPhoneVaild ? '9자 이상 가능합니다.(숫자만)' : '가능합니다!';
+    }
+    if (id === 'phoneCheck') {
+      return (
+        (codeUnderSix && '6자리를 입력해주세요.') ||
+        (codeOverTen && '10자리까지 가능합니다.')
+      );
+    }
+    if (id === 'email') {
+      return !isEmailVaild && '이메일 주소를 다시 확인해주세요.';
+    }
+  };
```

Comment:
```suggestion
  const inputMessageCondition = {
    phone: isPhoneVaild ? '가능합니다!' : '9자 이상 가능합니다.(숫자만)',
    phoneCheck:
      (codeUnderSix && '6자리를 입력해주세요.') ||
      (codeOverTen && '10자리까지 가능합니다.'),
    email: !isEmailVaild && '이메일 주소를 다시 확인해주세요.',
  };
```
어제 얘기 나눴던 부분이네요!
이렇게 반영해주시고 왜 이렇게 될지는 한번 확인하시고 얘기해 보시죠!